rust/flatbuffers/src/vector.rs

```diff
@@ -28,6 +28,14 @@ use primitives::*;
 #[derive(Debug)]
 pub struct Vector<'a, T: 'a>(&'a [u8], usize, PhantomData<T>);
 
+impl<'a, T: Follow<'a> + 'a> Clone for Vector<'a, T> {
+    fn clone(&self) -> Self {
+        Vector(self.0, self.1, self.2)
+    }
+}
+
+impl<'a, T: Follow<'a> + 'a> Copy for Vector<'a, T> {}
```

Comment:
This follow bound does not seem necessary.

Reply:
Done.